### DISCUSSION TITLE
posts/Chinese-input-method/

### DISCUSSION THREAD

**Discussion by giscus** [legendPerceptor/legendPerceptor.github.io#3] [General]
# posts/Chinese-input-method/

如果你很早之前就开始使用Ubuntu系统，你可能安装了搜狗输入法，因为以前系统自带的输入法总是有各种各样奇怪的问题，而且安装起来特别麻烦，装好了还莫名奇妙不能用。这篇文章就带你彻底搞定在Ubuntu系统下输入法的问题。

https://www.yuanjianliu.net/posts/Chinese-input-method/

**Comment by yyyqwq** [comment]
请问博主有遇到过这样的问题嘛，ub使用系统中文输入法的时候，vscode只能选中光标所在行，而无法像win下可以用鼠标选中代码块。搜了一下网上的解决方法是切到系统英文输入法，然后就能选中了。:(
